src/version.cr

```diff
@@ -1,5 +1,5 @@
 module Shards
-  VERSION    = {{ `cat VERSION`.stringify.chomp }}
+  VERSION    = {{ `cat #{__DIR__}/../VERSION`.stringify.chomp }}
   BUILD_SHA1 = {{ `git log --format=%h -n 1 2>/dev/null || echo ""`.stringify.chomp }}
```

Comment:
You should be able to do `git -C #{__DIR__}/.. .....`

Reply:
Yes but that won't work when shards lives in a plain `lib` directory and doesn't have its git repo around.